app/questions/services/mailchimp.py

```diff
@@ -0,0 +1,59 @@
+from mailchimp3 import MailChimp
+
+from django.conf import settings
+from django.utils import timezone
+
+from questions.models import Submission
+
+
+def remind_incomplete():
+    """master function finding clients and sending reminder emails to them"""
+
+    # find COVID clients and send reminder
+    covid_clients = find_clients(topic='COVID')
+    send_email(clients=covid_clients, list_id='', workflow_id='', email_id='')
```

Comment:
We have a staging server as well as prod, so we will need a way to specify different list / workflow / email IDs between dev/prod/staging. I recommend doing this in the dev/prod/staging settings files, so that you can import a single setting.

Reply:
done